File: ex3/task3.4/YearsBeforeRetirement.java
/**
* Prints 3 lines of text with 3 variables
* Users Age is a comand line argument 
* For testing I have used the next command line arguments: Ginica, 1.23, 314, 20
* First 2 crashed the program
* The third one gave me an inappropiately result
* @author Vlad Georgescu 
*/
public class YearsBeforeRetirement
{
    public static void main(String[] args)
    {
        int myAgeNow;
        int retirementAge;
        int yearsBeforeRetirement;
        myAgeNow = Integer.parseInt(args[0]);
        retirementAge = 65;
        yearsBeforeRetirement = retirementAge - myAgeNow;
        System.out.println("I am " + myAgeNow);
        System.out.println("Retirement in Romania is at " + retirementAge);
        System.out.println("I have to work " + yearsBeforeRetirement + " years from now on");
    }
}

File: Ex3/Task3.4/YearsBeforeRetirement.scala
/**
 * Shows how many years I have left before retirement
 * The age is passed as a command line argument
 */
object YearsBeforeRetirement {
    def main(args: Array[String]) {
        var myAge: Int = args(0).toInt
        var retirementAge: Int = 68
        var yearsLeft: Int = retirementAge - myAge

        println("My age now is " + myAge)
        println("I will retire at the age of " + retirementAge)
        println("Years left working is " + yearsLeft)
    }
}